svm.py

```diff
@@ -0,0 +1,73 @@
+#%%
+from sklearn.svm import SVR, LinearSVR
+from sklearn.model_selection import GridSearchCV
+from sklearn import preprocessing
+from sklearn import datasets
+import numpy as np
+dataset = datasets.load_boston()
+from sklearn import model_selection
+from sklearn import preprocessing
+X = dataset['data']
+y = dataset['target']
+#%%
+class Svm_svr:
+    
+    def __init__(self) -> None:
+       
+        self.param_grid = {
+                    #'C': np.logspace(-3, 2, 6), 
+                    'kernel': ['linear'],
+                    #'degree': np.logspace(2, 3, 4),
+                    #'gamma' : np.logspace(-3, 2, 6)
+                    'epsilon': [ 0.5, 0.1, 1.5]
+                    } 
+        self.clf = GridSearchCV(SVR(), self.param_grid ,cv=5, scoring='accuracy')
+        
+
+    def fit(self, dataset):
+        X = dataset['data']
+        y = dataset['target']
+        self.clf.fit(X, y)
+        best_parameters = self.clf.best_params_
+        final_model = self.clf.best_estimator_
+    
+    def score_all():
+        final_model.score(train), final_model.score(test), final_model.score(val)
+       
+
+#%%
+model = Svm_svr()
```

Comment:
We don't want to initialise the model in this file. Only in main 

or  can add it under
```
if __name__ == '__main__':
    ...
```